### DISCUSSION TITLE
Update to php 8.1.4

### DISCUSSION THREAD

**Discussion by UlrichHP** [oerdnj/deb.sury.org#1757] [Q&A] [answered]
Hi,

On my computer, php 8.1.2 is installed with this ppa but apt update doesn't show an update to php 8.1.4.

Is it normal ?
I couldn't find any infos about it elsewhere.

Thank you

**Comment by oerdnj** [comment]
Your question doesn't contain any information at all that could be used to help you. We can't really work with "it's broken"...

**Reply by UlrichHP** [reply]
Oh sure, sorry.

My computer is running Ubuntu 21.04 with KDE and i followed a tutorial to install `ppa:ondrej/php`
When php 8.1.2 was released, when doing `sudo apt update`, i had the option to upgrade php 8.1.1 to 8.1.2

But not anymore, it doesn't show me a way to upgrade to 8.1.4 with `apt update`.

**Reply by UlrichHP** [reply]
Could it be blocked by a module not yet compatible with php 8.1.4 ?

**Comment by mxdpeep** [comment, marked as answer]
it may be related to the fact, that you are using a deprecated OS - Ubuntu
21.04 lasts only 1 year, you should always use even LTS versions like 18,
20, 22

21.04 and 21.10 are testing versions of Canonical Ubuntu FYI

**Reply by UlrichHP** [reply]
You were right, i upgraded to Ubuntu 21.10 and i now have php 8.1.6.

Thanks
